userappwithbroker/src/main/java/com/microsoft/aad/adal/example/userappwithbroker/Constants.java

```diff
@@ -108,4 +110,20 @@ public String getText() {
             return text;
         }
     }
+
+    enum AssertionVersion {
+        None(null),
+        Version1_1(AuthenticationConstants.OAuth2.MSID_OAUTH2_SAML11_BEARER_VALUE),
+        Version2(AuthenticationConstants.OAuth2.MSID_OAUTH2_SAML2_BEARER_VALUE);
+
+        private final String text;
```

Comment:
This variable name doesn't work for me; what is this value? Please javadoc, rename getters/setters

Reply:
Hey @iambmelt, Sorry about that. Was following the same way all the other constants were structured in that file. I'll rename it.